tests/test_client.py

```diff
@@ -138,30 +139,6 @@ def test_step002_ingest_events(self):
             res.ingested == 2
         ), f"expected ingested count to equal 2, found {res.ingested}"
 
-    def test_step003_ingest_wrong_encoding(self):
```

Comment:
removed those since the 2 fields are mandatory and python throws an error if they are not passed.